backend/src/api/routes/status.py

```
@@ -34,10 +34,15 @@ async def get_processing_status(
         # Validate trip ID format
         validated_trip_id = validate_trip_id(trip_id)
 
-        logger.info(f"Status check requested", extra={"trip_id": validated_trip_id})
+        logger.info(
+            f"Status check requested", 
```

Ruff (0.13.1)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)